src/Modules/Extensions/templates/Backend/Actions/ThemeDetail.html.twig

```diff
@@ -132,10 +132,10 @@
             <p>{{ 'msg.ConfirmThemeInstall'|trans }}</p>
           </div>
           <div class="modal-footer">
-            {{ macro.buttonIcon('', 'times', 'lbl.No'|trans|ucfirst, 'btn-default', { "data-bs-dismiss":"modal", "type":"button" }) }}
-            {{ form_start(themeInstallForm) }}
-            {{ macro.buttonIcon(null, 'check', 'lbl.Yes'|trans|ucfirst, 'btn-primary', { "type":"submit"}) }}
-            {{ form_end(themeInstallForm) }}
+            {{ macro.buttonIcon('', 'times', 'lbl.No'|trans|ucfirst, 'btn-default', {"data-bs-dismiss": "modal", "type": "button"}) }}
+            {{ form_start(theme_install_form) }}
```

Comment:
Should't there be also a change in the PHP where `theme_install_form` is passed to the template?

Reply:
@bjorvack is this fixed/checked?

Reply:
fixed bcd6511c6b18bd62f9c0685cfd4b3a9674b6e7e9